### ISSUE TITLE
How to start a new Xcode Project using Commandant?

### ISSUE THREAD

**Issue by alvivi** [Carthage/Commandant#28]
I want to create new Xcode project to build a CLI tool. So I open Xcode, I create a new command line tool project (using swift), then I try to add Commandant as a dependency, but, I cannot add frameworks as a dependency.

All the projects using Commandant have very peculiar xcodeproj. They seem standard OSX apps, but they are using build phases like "Extract CLI Tool".

Am I missing a tool or a bootstrap project?

**Comment by mdiep** [comment]
You have to jump through some hoops to use Swift in a CLI tool. I'd try to copy what Carthage does.

**Comment by neilpa** [comment]
The root of the problem is that CLI tools statically link the swift stdlib while frameworks dynamically link it. AFAIK there's no way to force dynamic linkage of the stdlib for non-app bundles (c.f. Carthage/Carthage#16).

**Comment by Danappelxx** [comment]
Can anyone create a summary of how to use Commandant in a CLI Xcode project (and put it in the README)? Sure, I could look through Carthage's source, but there's a lot more to Carthage than just Commandant, so I would be wasting my time looking through it.
Thanks

**Comment by griffin-stewie** [comment]
I agree with @Danappelxx . I need some simple template project for creating CLI tool with Commandant. Carthage it self is too complicated as starting point to learn how to use Commandant.

**Comment by mdiep** [comment]
This would be a great thing to add to the Commandant repo. PRs welcome!

**Comment by vknabel** [comment]
The easiest way is probably using the SwiftPM and generating the xcodeproj. 

`$ swift package init --type executable`
Then adding Commandant as dependency. 
Art the end just run
`$ swift package generate-xcodeproj`

**Comment by griffin-stewie** [comment]
Thank you @vknabel . It works!
